database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,11 +14,8 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
+            $table->bigIncrements('id');
```

Comment:
We should use the `->id()` here. Under the hood it is just calling `bigIncrements('id')`

Reply:
Oh, I never knew that. I'll revert it back.